src/node/nodetonode.h

```diff
@@ -50,7 +50,8 @@ namespace ccf
     }
 
     template <class T>
-    void send_authenticated(NodeId to, const T& data)
+    void send_authenticated(
```

Comment:
Minor: My preference would be to have `msg_type` as the first argument to that function.

Reply:
Minor: `NodeMsgType msg_type` is fine, rather than `const&`